### ISSUE TITLE
Allowing Custom Formatting of labels.inner.format and labels.outer.format

### ISSUE THREAD

**Issue by wgibbs123** [benkeen/d3pie#55]
I currently use "label" as the labels.outer.format for my D3PIE charts and "value" as the labels.inner.format.
This works, but would be nice if we could add custom formatting.  For instance if my labels.inner.format is a dollar amount maybe I want to display $1,999,999.00 over 1999999.  I don't believe this is possible right now without customizing the D3 API directly.

**Comment by benkeen** [comment]
Agreed, this would be a really handy option. Tooltips have just this feature, but not the inner/outer labels. I'll include a function for this in the next release (0.1.7 - I'll release 0.1.6 today).

**Comment by wgibbs123** [comment]
Thank you for this and the hard work, greatly appreciated

**Comment by dagostinelli** [comment]
+1

I stumbled across this library and experimentally attempted to use it.  As soon as I asked "hmm, how do I add the dollar sign to these numbers", I found out that I couldn't.  This is probably going to be a showstopper for my project.
